test/custom.test.ts

```diff
@@ -325,4 +329,37 @@ describe("Custom Contracts", async () => {
     expect(nfts.length).to.eq(1);
     expect(nfts[0].metadata.name).to.eq("Custom NFT");
   });
+
+  it("should detect feature: erc721 signature mintable", async () => {
+    const c = await sdk.getContractFromAbi(
+      sigDropContractAddress,
+      SignatureDrop__factory.abi,
+    );
+
+    invariant(c, "Contract undefined");
+    invariant(c.nft, "ERC721 undefined");
+    invariant(c.nft.signature, "ERC721 drop");
+
+    const payload = {
+      metadata: {
+        name: "OUCH VOUCH",
+      },
+      to: samWallet.address, // Who will receive the NFT (or AddressZero for anyone)
+      price: 0.5, // the price to pay for minting
+      currencyAddress: NATIVE_TOKEN_ADDRESS, // the currency to pay with
+      royaltyBps: 100, // custom royalty fees for this NFT (in bps)
+      quantity: "1",
+    };
+
+    let goodPayload: SignedPayload721WithQuantitySignature;
+
+    goodPayload = await c.nft.signature.generate(payload);
+
+    const valid = await c.nft.signature.verify(goodPayload);
```

Comment:
make sure both SigDrop and NFTCollection sigmint tests pass - might need to adapt the common class to handle both